simple-git-hooks.js

```diff
@@ -178,7 +178,13 @@ function _setHook(hook, command, projectRoot=process.cwd()) {
         return
     }
 
-    const hookCommand = "#!/bin/sh\n" + command
+    const prependScript =
```

Comment:
I think its better to export this as const, and use in tests, so we avoid code repeating